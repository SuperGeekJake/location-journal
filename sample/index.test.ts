import { describe, it, expect } from "vitest";
import * as data from "./index";

describe("Fabricated data", () => {
  it("visits should match snapshot", () => {
    expect(data.sampleCSV).toMatchSnapshot();
  });
});
