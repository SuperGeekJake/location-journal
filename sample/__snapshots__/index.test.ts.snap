// Vitest Snapshot v1

exports[`Fabricated data > visits should match snapshot 1`] = `
"Date,Location Name,Image Url,Featured,Description
2022-04-13,Ground Up Cafe,https://placekitten.com/150/150?image=7,false,Best matcha latte!
2022-04-29,Wide Awake Cafe,https://placekitten.com/150/150?image=9,false,Just okay
2022-04-30,Jumpin' Beans Cafe,https://placekitten.com/150/150?image=8,false,Best matcha latte!
2022-05-04,The Split Bean,https://placekitten.com/150/150?image=14,false,Stale croissants only
2022-05-11,Tatiana's Cafe,https://placekitten.com/150/150?image=4,false,Love the atmosphere
2022-05-18,Impresso Espresso,https://placekitten.com/150/150?image=1,true,Worst matcha latte :(
2022-05-25,Espresso Express,https://placekitten.com/150/150?image=1,false,Best coffee in town
2022-05-26,Expresso,https://placekitten.com/150/150?image=5,false,Pistachio croissant Yum!
2022-05-26,Espresso Cafe,https://placekitten.com/150/150?image=15,true,Great service
2022-05-28,Baristas,https://placekitten.com/150/150?image=6,true,Great service
2022-06-04,Jumpstart Coffee Shop,https://placekitten.com/150/150?image=3,false,Love the decor
2022-06-11,The Roasted Bean,https://placekitten.com/150/150?image=11,false,Love the decor
2022-06-13,Espresso Express,https://placekitten.com/150/150?image=15,false,Love the decor
2022-06-13,The Family Bean,https://placekitten.com/150/150?image=5,false,Coffee is too hot
2022-06-13,Wake Up Cafe,https://placekitten.com/150/150?image=9,true,Stale croissants only
2022-06-17,Beany Business,https://placekitten.com/150/150?image=7,true,Just okay
2022-06-19,Tatiana's Cafe,https://placekitten.com/150/150?image=2,false,Best coffee in town
2022-06-22,Spiced Chai Cafe,https://placekitten.com/150/150?image=8,true,Love the atmosphere
2022-06-23,Boston Barista,https://placekitten.com/150/150?image=3,true,Best coffee in town
2022-06-26,Dream Bean Coffee Shop,https://placekitten.com/150/150?image=13,false,Just okay
2022-07-02,Mugs Coffee,https://placekitten.com/150/150?image=10,false,Worst matcha latte :(
2022-07-12,Baristas,https://placekitten.com/150/150?image=8,false,Worst matcha latte :(
2022-07-12,Baristas,https://placekitten.com/150/150?image=3,false,Best matcha latte!
2022-07-13,Grind House,https://placekitten.com/150/150?image=8,false,Stale croissants only
2022-07-13,Spiced Chai Cafe,https://placekitten.com/150/150?image=11,false,Just okay
2022-07-13,Lava Java,https://placekitten.com/150/150?image=2,false,Pistachio croissant Yum!
2022-07-17,The Hideout,https://placekitten.com/150/150?image=16,false,Pistachio croissant Yum!
2022-07-21,Espresso Express,https://placekitten.com/150/150?image=13,false,Best matcha latte!
2022-07-22,Jumpstart Coffee Shop,https://placekitten.com/150/150?image=8,false,Worst matcha latte :(
2022-07-22,Aroma Mocha,https://placekitten.com/150/150?image=4,false,Just okay
2022-07-27,Boston Barista,https://placekitten.com/150/150?image=4,false,Best matcha latte!
2022-07-27,Jumpstart Coffee Shop,https://placekitten.com/150/150?image=6,true,Love the decor
2022-07-27,Screamin' Beans,https://placekitten.com/150/150?image=6,true,Love the atmosphere
2022-07-27,Ground Up Cafe,https://placekitten.com/150/150?image=9,false,Stale croissants only
2022-07-27,Steamin' Mugs,https://placekitten.com/150/150?image=2,false,Too loud
2022-07-28,Impresso Espresso,https://placekitten.com/150/150?image=12,true,Pistachio croissant Yum!
2022-07-28,Beans 'n Cream Cafe,https://placekitten.com/150/150?image=14,false,Just okay
2022-07-29,Beany Business,https://placekitten.com/150/150?image=9,false,Love the atmosphere
2022-07-30,Grind House,https://placekitten.com/150/150?image=1,true,Great service
2022-07-31,The Friendly Bean,https://placekitten.com/150/150?image=4,false,Great service
2022-08-02,Expresso,https://placekitten.com/150/150?image=1,false,Love the atmosphere
2022-08-05,Cup o' Joe,https://placekitten.com/150/150?image=2,false,Best matcha latte!
2022-08-05,Steamin' Mugs,https://placekitten.com/150/150?image=6,false,Worst matcha latte :(
2022-08-07,Espresso Express,https://placekitten.com/150/150?image=14,false,Best coffee in town
2022-08-12,The Grind,https://placekitten.com/150/150?image=1,false,Best coffee in town
2022-08-12,Espresso Cafe,https://placekitten.com/150/150?image=16,false,Just okay
2022-08-14,Cup o' Joe,https://placekitten.com/150/150?image=4,true,Worst matcha latte :(
2022-08-16,The Roasted Bean,https://placekitten.com/150/150?image=1,true,Love the decor
2022-08-16,Coffee House,https://placekitten.com/150/150?image=3,false,Worst matcha latte :(
2022-08-18,Grinders Cafe,https://placekitten.com/150/150?image=10,true,Love the decor
2022-08-22,HuggaMug Cafe,https://placekitten.com/150/150?image=12,false,Worst matcha latte :(
2022-08-22,Dream Bean Coffee Shop,https://placekitten.com/150/150?image=2,true,Just okay
2022-08-23,The Grind,https://placekitten.com/150/150?image=10,false,Too loud
2022-08-26,Lava Java,https://placekitten.com/150/150?image=6,false,Pistachio croissant Yum!
2022-08-26,Spiced Chai Cafe,https://placekitten.com/150/150?image=7,true,Best coffee in town
2022-08-28,Topped Off,https://placekitten.com/150/150?image=13,false,Pistachio croissant Yum!
2022-08-28,The Family Bean,https://placekitten.com/150/150?image=15,false,Best coffee in town
2022-08-28,Wake Up Cafe,https://placekitten.com/150/150?image=2,false,Just okay
2022-08-30,Impresso Espresso,https://placekitten.com/150/150?image=5,true,Love the atmosphere
2022-08-31,Beans 'n Cream Cafe,https://placekitten.com/150/150?image=11,false,Just okay
2022-09-01,The Friendly Bean,https://placekitten.com/150/150?image=2,true,Pistachio croissant Yum!
2022-09-02,Beany Business,https://placekitten.com/150/150?image=8,true,Love the atmosphere
2022-09-03,The Split Bean,https://placekitten.com/150/150?image=14,false,Best coffee in town
2022-09-03,Steam Beans Cafe,https://placekitten.com/150/150?image=8,false,Pistachio croissant Yum!
2022-09-06,Espresso Express,https://placekitten.com/150/150?image=2,false,Too loud
2022-09-06,Dream Bean Coffee Shop,https://placekitten.com/150/150?image=7,false,Just okay
2022-09-07,Expresso,https://placekitten.com/150/150?image=11,true,Coffee is too hot
2022-09-18,Topped Off,https://placekitten.com/150/150?image=3,false,Stale croissants only
2022-09-19,The Split Bean,https://placekitten.com/150/150?image=8,false,Best matcha latte!
2022-09-20,Jumpstart Coffee Shop,https://placekitten.com/150/150?image=15,false,Best matcha latte!
2022-09-22,Boston Barista,https://placekitten.com/150/150?image=10,false,Great service
2022-09-23,The Coffee Club,https://placekitten.com/150/150?image=11,false,Worst matcha latte :(
2022-09-25,Cup o' Joe,https://placekitten.com/150/150?image=5,false,Great service
2022-09-26,Beany Business,https://placekitten.com/150/150?image=2,false,Love the decor
2022-09-26,Steam Beans Cafe,https://placekitten.com/150/150?image=6,false,Pistachio croissant Yum!
2022-09-28,Coffee House,https://placekitten.com/150/150?image=10,false,Worst matcha latte :(
2022-10-01,Beany Business,https://placekitten.com/150/150?image=10,false,Pistachio croissant Yum!
2022-10-03,Cup o' Joe,https://placekitten.com/150/150?image=6,false,Love the atmosphere
2022-10-03,Espresso Express,https://placekitten.com/150/150?image=11,false,Worst matcha latte :(
2022-10-06,Jumpstart Coffee Shop,https://placekitten.com/150/150?image=13,false,Best matcha latte!
2022-10-06,Impresso Espresso,https://placekitten.com/150/150?image=13,false,Great service
2022-10-06,The Grind,https://placekitten.com/150/150?image=3,true,Stale croissants only
2022-10-07,Topped Off,https://placekitten.com/150/150?image=13,false,Pistachio croissant Yum!
2022-10-07,Wake Up Cafe,https://placekitten.com/150/150?image=16,true,Coffee is too hot
2022-10-08,The Roasted Bean,https://placekitten.com/150/150?image=12,true,Too loud
2022-10-08,Topped Off,https://placekitten.com/150/150?image=15,false,Best matcha latte!
2022-10-09,The Coffee Club,https://placekitten.com/150/150?image=12,false,Coffee is too hot
2022-10-10,Dream Bean Coffee Shop,https://placekitten.com/150/150?image=5,false,Love the atmosphere
2022-10-11,Wide Awake Cafe,https://placekitten.com/150/150?image=15,true,Great service
2022-10-15,The Coffee Club,https://placekitten.com/150/150?image=7,true,Stale croissants only
2022-10-17,Coffee Time,https://placekitten.com/150/150?image=13,false,Love the atmosphere
2022-10-21,Coffee Time,https://placekitten.com/150/150?image=12,true,Coffee is too hot
2022-10-21,Wake Up Cafe,https://placekitten.com/150/150?image=7,false,Coffee is too hot
2022-10-21,HuggaMug Cafe,https://placekitten.com/150/150?image=2,true,Love the decor
2022-10-22,Jumpin' Beans Cafe,https://placekitten.com/150/150?image=3,true,Just okay
2022-10-22,Baristas,https://placekitten.com/150/150?image=10,false,Pistachio croissant Yum!
2022-10-25,Steam Beans Cafe,https://placekitten.com/150/150?image=8,false,Love the decor
2022-10-27,Jacked Up Coffee,https://placekitten.com/150/150?image=16,true,Love the atmosphere
2022-10-29,Boston Barista,https://placekitten.com/150/150?image=16,false,Love the atmosphere
2022-10-29,Spiced Chai Cafe,https://placekitten.com/150/150?image=7,false,Love the atmosphere"
`;
